examples/tfengine/modules/team.hcl

```diff
@@ -490,10 +490,30 @@ template "additional_iam_members" {
 
 # Kubernetes Terraform deployment. This should be deployed after the GKE Cluster has been deployed.
 template "kubernetes" {
```

Comment:
For this to be actually tested in the presubmit integration tests, you need to add `kubernetes` dir to the `dirsToDeploy` list [here](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/tests/tfengine/integration_test.go#L43). Once you do that, it will probably error out as the cluster is private and the presubmit cloud build trigger cannot deploy to it.

Reply:
I know, I'm not adding it here so that the persubmit tests won't fail. I manually ran the presubmit tests to make sure the template can be deployed. Once the private cluster changes are in, I'll include it in the list of dirs to be tests.